### PR TITLE
Cleanup and test debug

### PR DESCRIPTION
**Pull request by cschroed-usgs**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: r
+cache: packages
```
Comment (cschroed-usgs) [inline .travis.yml]: might speed up travis build

File: tests/testthat/helper_test_utils.R [hunk 2]
```diff
@@ -32,4 +32,10 @@ expect_has_names <- function(dataframe, names) {
 		error_message <- paste('expected names', expected_names, '\nactual names', actual_names, sep = "\n")
 		fail(error_message)
 	}
+}
+
```
Comment (cschroed-usgs) [inline tests/testthat/helper_test_utils.R]: @supercasey: I put your sweet concise sigfig counting functionality in this descriptively-named function and updated usages throughout the tests.

File: tests/testthat/test_may_load.R [hunk 3]
```diff
@@ -81,24 +81,22 @@ test_that("may loads are less than corresponding annual loads for a given site/w
 
 
 test_that("Most recent water year has all of the necessary sites ", {
-  result <- validate::check_that(temp_mloads_recent,
```
Comment (cschroed-usgs) [inline tests/testthat/test_may_load.R]: This new way of testing will provide more helpful info on failure.

**Comment by cschroed-usgs** [comment]
The aloads and mloads data frames have been re-saved with compress='xz'. They were not otherwise modified.

File: tests/testthat/test_may_flow.R [hunk 4]
```diff
@@ -56,19 +56,16 @@ test_that("may flow is less than corresponding annual flows for a given site/wat
 })
 
 test_that("Most recent water year has all of the necessary sites ", {
```
Comment (cschroed-usgs) [inline tests/testthat/test_may_flow.R]: This new way of testing will provide more helpful info on failure. Site abbreviation lists are identical.

File: tests/testthat/test_annual_load.R [hunk 5]
```diff
@@ -73,23 +73,21 @@ test_that("annual loads for the GULF are included", {
 })
```
Comment (cschroed-usgs) [inline tests/testthat/test_annual_load.R]: This new way of testing will provide more helpful info on failure. Site abbreviation lists are identical.

File: tests/testthat/test_annual_flow.R [hunk 6]
```diff
@@ -44,7 +44,7 @@ test_that("annual flow has a reasonable range of values", {
 
 test_that("Flow data have the correct number of significant digits", {
   result <- validate::check_that(aflow, 
-                                 nchar(sub("^[0]+", "",sub("[.]","",aflow$FLOW/1E11))) <= 3
+                                 nchar(aflow$FLOW/1E11) <= 3
```
Comment (cschroed-usgs) [inline tests/testthat/test_annual_flow.R]: oops, that should be count_sig_figs(), not nchar()

**Comment by cschroed-usgs** [comment]
Travis build is passing now. @supercasey, what do you think of these changes?

**Comment by supercasey** [comment]
They look good to me, thanks.

**Pull request merged**